docs/api-types/issuer/spec.md

```diff
@@ -127,6 +127,7 @@ clouddns:
 route53:
   accessKeyID: AKIAIOSFODNN7EXAMPLE
   region: eu-west-1
+  hostedZoneID: DIKER8JPL21PSA
```

Comment:
Can we leave this bit out here? I'd rather not encourage people to set this field. It is likely to be removed pending discussion in https://github.com/jetstack-experimental/cert-manager/issues/94